### PR TITLE
Added form submit success/failure panels.

### LINKED ISSUE #14: We need a submission confirmation screen!
I have definitely submitted the form multiple times in a row because there's no indication that it's submitted—it just sits there looking sad.

### PR DESCRIPTION
**Pull request by danielwrobert**
Placeholder text added for each panel. Adjust as needed.

Fixes #14, when merged.

### CODE REVIEW

File: src/components/SubmitSuccess.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+			<CSSTransitionGroup
+				transitionName="step"
+				transitionEnterTimeout={ 500 }
+				transitionLeaveTimeout={ 500 }
+				transitionAppear={ true }
+				transitionAppearTimeout={ 500 }>
+
+				<div className="panel-intro">
+					<p>Success!</p>
+					<p>Your form has been submitted and your site is queued to be built by our team of evil, flesh-eating elves.</p>
```
Comment (sarahmonster) [inline src/components/SubmitSuccess.js]: I'm going to make the executive decision to remove "evil, flesh-eating elves" in the hopes of NOT giving everyone terrible nightmares.

**Comment by danielwrobert** [comment]
> If it's an unavoidable delay, we may want to indicate that visually.

**Comment by danielwrobert** [comment]
I am happy to refactor some things a bit but I suggest we do that in a separate Issue/PR to keep things smaller and focused.

For this, I will add the visual indication of the submit and, if that looks good to you, we can merge. From there we can open up a new ticket for the refactoring. Whadayasay?

**Comment by sarahmonster** [comment]
👍 Works by me!

**Comment by danielwrobert** [comment]
Found an error - don't merge yet! :)

**Pull request merged**